From 4efa24cc3faec0183689b0d477101aa5296e3c68 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 3 Apr 2018 20:49:42 +0800
Subject: [PATCH] drm/rockchip: rgb: init connector->port

this will be used by:
	rockchip_drm_bind()
	    ->show_loader_logo()
		->of_parse_display_resource()
		    ->find_connector_by_node()

Change-Id: I953c06d291a9475b7a60882039c3fc0e26959e46
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/rockchip/rockchip_rgb.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/rockchip/rockchip_rgb.c b/drivers/gpu/drm/rockchip/rockchip_rgb.c
index 5b8332179b7c..26ac585ba666 100644
--- a/drivers/gpu/drm/rockchip/rockchip_rgb.c
+++ b/drivers/gpu/drm/rockchip/rockchip_rgb.c
@@ -287,6 +287,7 @@ static int rockchip_rgb_bind(struct device *dev, struct device *master,
 				      "failed to attach panel: %d\n", ret);
 			goto err_free_connector;
 		}
+		connector->port = dev->of_node;
 	} else {
 		rgb->bridge->encoder = encoder;
 		ret = drm_bridge_attach(drm_dev, rgb->bridge);
-- 
2.35.3

